### PR TITLE
Bugfix & reformat context menu

### PR DESCRIPTION
**Pull request by Revadike**
It wasn't working properly. It _should_ be fixed now. I also did some reformatting

### CODE REVIEW

File: js/background/background.js [hunk 1]
```diff
@@ -514,50 +514,39 @@ ITAD_Api._progressingRequests = new Map();
...
-        for (let option of options) {
+        for (let option of Object.keys(ContextMenu.queryLinks)) {
             if (!SyncedStorage.get(option)) { continue; }
 
             browser.contextMenus.create({
                 "id": option,
                 "title": Localization.str.options[option].replace("__query__", "%s"),
                 "contexts": ["selection"]
-            });
+            }, () => chrome.runtime.lastError); // https://stackoverflow.com/q/54504328
```
Comment (MxtOUT) [inline js/background/background.js]: Why is this a duplicate ID in the first place?
Can't it be avoided?

**Comment by Revadike** [comment]
Pff, why do you always ask me these things weeks after the fact. 
The issue was that content menu option didn't open a new tab when clicking on it.
This fixes it.

**Comment by MxtOUT** [comment]
I asked for the cause, if you'd had included it directly, this could've been already merged

**Comment by MxtOUT** [comment]
The problem was that calls to `ContextMenu.update` from core.js weren't `await`ing `Localization` therefore it led to an error during the creation of the context menu entry

**Pull request merged**